packages/node_modules/@webex/internal-plugin-wdm/test/integration/spec/ping.js

```diff
@@ -1,121 +0,0 @@
-/*!
```

Comment:
The tests this file ran were outside of the scope of the JS SDK as a whole.

Reply:
Thanks. Not sure why we were tasked with making sure every service responds to pings.